packages/base-controller/src/index.ts

```diff
@@ -0,0 +1,17 @@
+export {
+  BaseConfig,
+  BaseController,
+  BaseState,
+  Listener,
+} from './BaseController';
+export {
+  BaseController as BaseControllerV2,
+  Listener as ListenerV2,
+  StateDeriver,
+  StateMetadata,
+  StatePropertyMetadata,
+  Json,
```

Comment:
Can we remove this and use the type from `@metamask/utils`?

Reply:
We certainly can, but I think we ought to do this in a separate PR.